batch_convert.qml

```diff
@@ -41,14 +42,16 @@ MuseScore {
         }
         else
             batchConvert.visible = true // needed for unknown reasons
+
+        if(settings.ePath===undefined) settings.ePath=settings.iPath;
```

Comment:
Formatting: space between `if` and `(`, nweline and 4 space indent after the `)`